templates/slurm-beegfs.txt

```diff
@@ -110,7 +110,7 @@ Order = 10
         Label = Scheduler VM Type
         Description = The VM type for scheduler node
         ParameterType = Cloud.MachineType
-        DefaultValue = Standard_D12_v2
+        DefaultValue = Standard_D16_v4
```

Comment:
The D12 is 4 cores/28 GB for $.368/hr. The D16_v4 is 16 cores/64 GB for $.896/hr. Seems like the D4_v4 (4/16 for $.224/hr) would be better, or at least the D8 in the middle.